src/snowflake/snowpark/modin/plugin/_internal/frame.py

```diff
@@ -27,6 +27,7 @@
     ROW_POSITION_COLUMN_LABEL,
     append_columns,
     assert_duplicate_free,
```

Comment:
is it cache_result or cache? Let's maybe use one name throughout our code base, and if desired define an alias at the frontend layer.